k-distribution/tests/regression-new/set-symbolic-tests/union-38-spec.k.out

```diff
@@ -0,0 +1,17 @@
+  #Not ( {
+    X
+  #Equals
+    x
+  } )
+#And
+  <k>
+    unionResult ( X:MyId in ( SetItem ( X:MyId )
+    SetItem ( y ) ) |Set SetItem ( x ) ) ~> .
```

Comment:
Shouldn't this get simplified?

Reply:
Would be nice if it was, yes. We would need a rule like `rule X in ((SetItem(X) S) |Set _) => true ensures notBool (X in S)` to "see through the set union and discover item `X`

Reply:
:+1: Rules added, changing union-37 and union-38 (I also added `union-39` for the right-hand side of `|Set`)